documentation/resource_ard.md

```diff
@@ -23,15 +20,15 @@ end
 where
 
 - `:activate` activates the ARD agent
-- `:configure` configures the agent using the `kickstart` defaut commandline arguments.
+- `:configure` configures the agent using the `kickstart` default commandline arguments.
 
 The default `:configure` action is equivalent to the following
 **System Preferences > Sharing** settings:
 
 ![Sharing Preferences](sharing_preferences.png)
 
 The full syntax for all of the properties that are available to the **ard**
-resource is:
+resource are:
```

Comment:
This should be `is` since we're referring to the singular `syntax`, not the plural `properties`.

Reply:
You are listing all of the properties though. So you are also referring to the plural properties?

Reply:
Changed these back, thanks for the clarification.